doc/Makefile

```diff
@@ -130,23 +130,3 @@ coverage:
 	$(SPHINXBUILD) -b coverage $(ALLSPHINXOPTS) $(BUILDDIR)/coverage
 	@echo "Testing of coverage in the sources finished, look at the " \
 		  "results in $(BUILDDIR)/coverage/python.txt."
-
-install:
-	rm -rf _build/doctrees _build/mne_bids.github.io
-	# first clone the mne-tools/mne_bids.github.io repo because it may ask
-	# for password and we don't want to delay this long build in
-	# the middle of it
-	# --no-checkout just fetches the root folder without content
-	# --depth 1 is a speed optimization since we don't need the
-	# history prior to the last commit
-	# -b gh-pages fetches only the branch for the gh-pages
-	git clone -b gh-pages --single-branch --no-checkout --depth 1 https://github.com/mne-tools/mne-bids _build/mne_bids.github.io
-	touch _build/mne_bids.github.io/.nojekyll
-	make html
-	cd _build/ && \
-	cp -r html/* mne_bids.github.io && \
-	cd mne_bids.github.io && \
-	git add * && \
-	git add .nojekyll && \
-	git commit -a -m 'Make install' && \
-	git push
```

Comment:
any chance of modifying this instead of deleting the whole thing? I like it be able to push the docs manually instead of just through CircleCI.

Reply:
Yes, it'd be possible ... but what I came up with seems quite ugly --> and there are still some assumptions that need to be fulfilled in order for this to work. I added comments. Any idea on how to improve this?


```Bash
# assuming in mne_bids/doc directory ... otherwise, "make install" would not work
git checkout master

# assuming remote is configured as upstream
git pull upstream master

# make sure the local branch is aligned 
git reset --hard upstream/master

# Go back to top directory and use the main Makefile
cd ..
make build-doc

# Temporarily store HTML outputs
cp -r doc/_build/html ~/tmp_mne_bids_docs

# Get gh-pages branch and make sure it is up to date
# assuming that the gh-pages branch already exists locally
git checkout gh-pages
git pull upstream gh-pages
git reset --hard upstream/gh-pages

# remove current dev docs from gh-pages branch and replace with new ones
rm -rf dev/
cp -r ~/tmp_mne_bids_docs dev/
rm -rf ~/tmp_mne_bids_docs

# add and push
git add .
git commit -m "manual update of dev docs [skip ci]"
git push
```

Reply:
but maybe this can also be done in another PR, I am anxious to get this merged :-)

Reply:
okay fair enough! mental sanity is important during work from home :)